datafoundation/templates/storagecluster.go

```diff
@@ -27,10 +25,12 @@ import (
 // StorageClusterTemplate is the template that serves as the base for the storage clsuter deployed by the operator
 
 const (
-	OSDSizeInTiB            = 4
-	backingStorageClassName = "default-ocs-storage-class"
+	OSDSizeUpperBoundInTiBUnscaled = 4398046511104
+	backingStorageClassName        = "default-ocs-storage-class"
 )
 
+var OSDSizeUpperBoundInTib = resource.NewQuantity(OSDSizeUpperBoundInTiBUnscaled, resource.BinarySI)
+
```

Comment:
If we cannot do an actual cost, can we mitigate the chance that someone will change the var value from the outside by using a functions?

```suggestion
	backingStorageClassName        = "default-ocs-storage-class"
)

func GetOSDSizeUpperBoundInTib() resource.Quantity {
    return resource.NewQuantity(4398046511104, resource.BinarySI)
}
```

Reply:
Switched back to `MustParse` in function scope